flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/scalarfunctions/MathScalarFunctions.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.table.runtime.scalarfunctions
+
+/**
+  * All build-in math scalar functions.
+  */
+class MathScalarFunctions {}
+
+object MathScalarFunctions {
+  /**
+    * Returns the natural logarithm of "x".
+    */
+  def log(x: Double): Double = {
```

Comment:
Can't we just use the `Math.log` as the built in method? It would reduce complexity as we would not need a CallGenerator then.